WcaOnRails/app/models/committee.rb

```diff
@@ -0,0 +1,33 @@
+# frozen_string_literal: true
+
+class Committee < ActiveRecord::Base
+  has_many :teams
+  has_many :team_members, through: :teams
+  has_many :committee_positions
```

Comment:
The code doesn't show a delete button if the committee has a committee position or if it has teams but I think you're right, this should go here anyway.